dist/Makefile

```diff
@@ -1,57 +1,29 @@
-all: help
+RELEASE      := 0.$(shell date +%Y%m%d).$(shell git describe --always)
 
-GIT_ROOT := $(shell git rev-parse --show-toplevel)
+ifeq "$(VERSION)" ""
+$(error Please specify VERSION e.g. VERSION=2.0)
+endif
+
+TAG := $(shell git describe --tags)
 
-VERSION := 666.dev
-RELEASE := 0.$(shell date +%Y%m%d).$(shell git describe --always)
 VERSION_NAME := $(VERSION)-$(RELEASE)
 
-GO_VERSION = $(shell cat ../.go-version)
+ifneq "$(TAG)" "0"
+$(shell git tag $(VERSION_NAME))
+endif
+
+GORELEASER := goreleaser --rm-dist
+
+.PHONY: release
+release:
+	echo $(RELEASE) > MANAGER-RELEASE-FILE
+	echo $(VERSION) > MANAGER-VERSION-FILE
+	$(GORELEASER) --skip-publish --skip-validate
```

Comment:
What's happening here, a comment would be good. I suggested a rename but maybe just RELEASE and VERSION. Do we intend to keep the files in git?

Reply:
If the files are not used let's remove that.